src/com/cdd/bao/template/Vocabulary.java

```diff
@@ -374,7 +377,18 @@ private void loadLabels(File baseDir, File[] extra, Set<String> exclude) throws
 			float progress = (float)progressSize / totalSize;
 			synchronized (listeners) {for (Listener l : listeners) l.vocabLoadingProgress(this, progress);}
 		}
-	
+
+		// find remappings, as distinguished from reparenting via preferredParent
+		Property batRemapTo = model.createProperty(ModelSchema.PFX_BAT + "remapTo");
+		for (StmtIterator iter = model.listStatements(null, batRemapTo, (RDFNode)null); iter.hasNext();)
```

Comment:
I've recently found an option to convert an iterator to an iterable:
```
	private <T> Iterable<T> iteratorToIterable(Iterator<T> iterator)
	{
		return () -> iterator;
	}
```
With this simple function, you could write:
```
for (Statement stmt : iteratorToIterable(model.listStatements(null, batRemapTo, (RDFNode)null))
```
Or without a function:
```
for (Statement stmt : (Iterable<Statement>) () -> model.listStatements(null, batRemapTo, (RDFNode)null)) {
```
If you think this is an interesting alternative, we could move the function into the Util class.

Reply:
I'm going to defer to Alex on this, as logic is modeled after earlier code.

Reply:
That's cute; the current style has the benefit of being familiar & common, albeit ugly. So I guess no action just yet, but we could consider migrating all of those style of loops at some point.